mantidproject/mantid
# Plot Selector Widget - initial functionality

**Description of work**

This is an initial implementation of the work described in this design document - https://github.com/mantidproject/documents/blob/master/Design/Workbench/Plot-Selector/design-plot-selector.md. This PR is open to review the code structure and check the basic functionality. Other features will be added in further PRs.

Current functionality:
- List of plots shown (names are the figure titles)
- Plot list can be filtered via text box
- Plots can be closed via close button or delete key 

**To test:**

Build and start the workbench. Make lots of plots, try closing and opening them and check nothing breaks. Does filtering work as expected?

Code review - check the MVP implementation here is sensible.

Fixes #22477 .

**Release Notes** 
*Workbench release notes will be gathered together separately*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
